src/compas/robots/model/geometry.py

```diff
@@ -424,19 +182,21 @@ class MeshDescriptor(BaseShape):
         The mesh' filename.
     scale : [float, float, float]
         The scale factors of the mesh in the x-, y-, and z-direction.
-    geometry : :class:`compas.datastructures.Mesh`
-        The COMPAS geometry of this mesh.
+    meshes : list of :class:`compas.datastructures.Mesh`
+        List of COMPAS geometric meshes.
 
     Examples
     --------
     >>> m = MeshDescriptor('link.stl')
 
     """
 
-    def __init__(self, filename, scale='1.0 1.0 1.0'):
+    def __init__(self, filename, scale='1.0 1.0 1.0', **kwargs):
```

Comment:
the docstring seems to be out of sync with the actual params

Reply:
fixed https://github.com/compas-dev/compas/pull/983/commits/4e1cc183d0fe7424290f1e8c495e1ac47ac15e20